components/symbl_ai/actions/get-conversations/get-conversations.mjs

```diff
@@ -0,0 +1,81 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-get-conversations",
+  name: "Get Conversations",
+  description: "Get a list of all conversations. See the doc [here](https://docs.symbl.ai/docs/conversation-api/all-conversations/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    limit: {
+      type: "integer",
+      label: "Limit",
+      description: "Specifies the limit number of conversations to be returned. Default value is `20`.",
+      optional: true,
+      min: 0,
+      max: 65536,
+    },
+    offset: {
+      type: "integer",
+      label: "Offset",
+      description: "Specifies the number of items to skip before applying `limit`",
+      optional: true,
+    },
+    order: {
+      type: "string",
+      label: "Order",
+      description: "Specifies the order in which the results should be sorted based on the start time.",
+      options: [
+        "asc",
+        "desc",
+      ],
```

Comment:
In general, it is preferred to use a `./common/constants.mjs` file, something like this:

```javascript
./common/constants.mjs

const resultsOrder = {
  ASC: "asc",
  DESC: "desc",
};

export default {
  resultsOrder,
};
```

And then reference it:

```suggestion
      options: Object.values(constants.resultsOrder),
```